packages/http-client-csharp/emitter/src/options.ts

```diff
@@ -151,8 +143,6 @@ export const defaultOptions = {
   "new-project": false,
   "clear-output-folder": false,
   "save-inputs": false,
-  "generate-protocol-methods": true,
```

Comment:
I think we do support these as they are passed into the context that goes into the shouldGenerateProtocol TCGC call.